internal/multitenancy/multitenancy.go

```diff
@@ -157,8 +157,9 @@ func (tdm *TenancyDatamodel) setupProject(ctx context.Context, project *nexus.Ru
 
 	// Label default template
 	var defaultTemplateName string
+	baselineRegex := regexp.MustCompile(`^baseline-v\d+\.\d+\.\d+`)
 	for _, t := range tdm.templates {
-		if strings.Contains(t.GetName(), template.DefaultTemplateName) {
```

Comment:
can we delete this const completely? 
https://github.com/open-edge-platform/cluster-manager/blob/main/internal/template/templates.go#L20

Reply:
sure!